Source/Core/InputCommon/ControllerInterface/evdev/evdev.cpp

```diff
@@ -0,0 +1,270 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <libudev.h>
+#include <unistd.h>
+
+#include "Common/Logging/Log.h"
+#include "InputCommon/ControllerInterface/evdev/evdev.h"
+
+
+namespace ciface
+{
+namespace evdev
+{
+
+void Init(std::vector<Core::Device *> &controllerDevices)
```

Comment:
No space for `*` and ampersand against the type; this is present in other areas below as well.